templating/twig_extension_repository.rst

```diff
@@ -0,0 +1,93 @@
+.. index::
+   single: Using the Twig Extensions Repository
+
+How to Use the Twig Extensions Repository
```

Comment:
Can we change the title of this article?  The `Repository` word is confusing here. What we want is to enable/use the official Twig extensions in Symfony apps.